x/bank/keeper/send.go

```diff
@@ -24,8 +24,11 @@ type SendKeeper interface {
 	SetBalance(ctx sdk.Context, addr sdk.AccAddress, balance sdk.Coin) error
 	SetBalances(ctx sdk.Context, addr sdk.AccAddress, balances sdk.Coins) error
 
-	GetSendEnabled(ctx sdk.Context) bool
-	SetSendEnabled(ctx sdk.Context, enabled bool)
+	GetParams(ctx sdk.Context) types.Params
+	SetParams(ctx sdk.Context, params types.Params)
```

Comment:
Why did we add `SetParams` to `SendKeeper`? This breaks ocaps even more. Just noting that this should be placed elsewhere at some point in the future.